src/ui/sass/_mixins.scss

```diff
@@ -5,7 +5,6 @@
   $style: normal,
   $color: $color-text-primary
 ) {
-  font-family: $font-family;
```

Comment:
Just want to confirm, but removing this won't break anybody's custom styling, right? If someone built their site with template-answers, they could conceivably have used this mixin in any custom styling. They'd be fine though, even with this removed, since whichever class they'd have styled should be underneath a 'yxt-Container' div.

Reply:
Yes exactly! Most implementers styling that would depend on this will still work - e.g. changing the SDK's font to their custom one - because this makes the styling less specific

This would break custom styling if 

1. This mixin was used outside of an answers component container. I don't know what our recommendation would be here, what do you think?
2.  If an implementer was applying a custom font-family to the answers component container CSS selector. Implementers would need to ensure their styling was imported after the SDK styling - this is currently how it works in client implementations, so no change required.
3.  If an implementer was applying styling to a more-specific selector than the answers component container but expecting it **not** to work and instead have the SDK's font-family applied. This seems like a bad practice, it may be OK to break this if it's happening, what do you think?

Reply:
1. I don't think we need to guarantee correctness if they use Answers CSS for non-Answers HTML. That's not something we'd recommend anyone do.
2. Almost always, if you're trying to override a library's styling, you want your styling to be loaded afterwards. That's certainly how it's always worked overriding the SDK's styling.
3. That does seem like a bad practice. If they added a font-family to a more specific selector, they may have wanted it to work. Now, it will. If they didn't actually want that font-family and left it, they left broken/irrelevant CSS in their style sheets. They were depending on what's arguably a bug in the SDK. I don't think we'd want to support that. It seems pretty unlikely.

Reply:
Amazing, that sounds right to me!